cspell.config.json

```diff
@@ -0,0 +1,180 @@
+    "language": "en",
+    // list of patterns to be ignored
+    "ignoreRegExpList": [
+        // ignores method parameters like: '(IFFF)', '(DDDDDDFD)'
+        "/\\(.+\\)/g",
+        // ignores types like: 'Ljava/lang/String;'
+        "/L[A-Za-z0-9/_$]+;/g"
+    ],
+    // words - list of words to be always considered correct
+    "words": [
+        "alloc",
+        "attackable",
+        "authlib",
+        "BIOMESET",
+        "blit",
+        "BLOCKSTATE_PROPERTY",
+        "bossbar",
+        "breedable",
+        "CAERBANNOG_RABBIT",
+        "CHAINMAIL",
+        "CHESTPLATE",
+        "clazz",
+        "CLOWNFISH",
+        "collidable",
+        "concat",
+        "confiming",
+        "cooldown",
+        "craftable",
+        "CREEBET",
+        "datafix",
+        "datafixer",
+        "DEADBUSH",
+        "Deadmau5",
+        "Decryptor",
+        "deflater",
+        "deop",
+        "despawn",
+        "dont",
+        "DOWNLEFT",
+        "dpfail",
+        "dppass",
+        "drainable",
+        "elytra",
+        "enchantability",
+        "enchantable",
+        "Encryptor",
+        "ENDERCHEST",
+        "enderdragon",
+        "enderman",
+        "endermite",
+        "EPOLL",
+        "equipable",
+        "etho",
+        "fabricmc",
+        "flowable",
+        "FORCEFIELD",
+        "framebuffer",
+        "fullscreen",
+        "gamemode",
+        "gamerules",
+        "GLOWSTONE",
+        "gson",
+        "heightmap",
+        "hitbox",
+        "hitboxes",
+        "HOLDABLES",
+        "hotbar",
+        "hoverable",
+        "illager",
+        "infestable",
+        "ingame",
+        "ints",
+        "invisibles",
+        "itemable",
+        "itemgroup",
+        "kbps",
+        "keybind",
+        "knockback",
+        "lerp",
+        "lfloor",
+        "lightmap",
+        "lwjgl",
+        "minecart",
+        "minecarts",
+        "mineshafts",
+        "mipmap",
+        "MIPPED",
+        "mojang",
+        "mooshroom",
+        "mtick",
+        "neighbour",
+        "neighbours",
+        "NETHERBRICK",
+        "NETHERRACK",
+        "NIGHTVISION",
+        "NOTEBLOCK",
+        "npcs",
+        "occulsion",
+        "opengl",
+        "overworld",
+        "PAEONIA",
+        "pathed",
+        "pathfinding",
+        "pathing",
+        "PerlinNoise",
+        "pigman",
+        "PIGSCENE",
+        "PODZOL",
+        "Populator",
+        "PORKCHOP",
+        "pottable",
+        "Prepender",
+        "println",
+        "PRISMARINE",
+        "Programi",
+        "PUFFERFISH",
+        "PURPUR",
+        "pushable",
+        "randomizer",
+        "raycast",
+        "rcon",
+        "RECORD_MELLOHI",
+        "RECORD_STAL",
+        "RECORD_STRAD",
+        "regen",
+        "reloadable",
+        "renderable",
+        "renderbuffer",
+        "resourcepack",
+        "respawn",
+        "rgba",
+        "rtmp",
+        "schoolable",
+        "ScreenHandlerProprtyUpdateS2CPacket",
```

Comment:
Proprty?

Reply:
Problem is that currently there are two classes with the same name: `ScreenHandlerProprtyUpdateS2CPacket` and  `ScreenHandlerPropertyUpdateS2CPacket`.
That's why I couldn't fix the spelling and added it to the ignore list.

Reply:
The new system allows having multiple class mappings in the same file so you can probably move the misspelled one in the correct file and rename the mapping at the same time.

Reply:
I didn't know that this was possible